content/post/ds_advice.md

```diff
@@ -0,0 +1,59 @@
+
+<h3>Basic stats</h3>
+
+Learn linear regression, hypothesis testing and logistic regression. These topics are foundational and complex enough to keep you busy for a while, especially if you don't have a strong Statistics background. 
+
+To the point above about learning the overlap - understand that linear regression can be formulated in multiple ways, each with its own nuance. For example, formulating LR as a statistical problem involves thinking probabilistically, talking about residuals and making assumptions around independence, homoscedasticity and Normality. It also involves minimizing a log likelihood function and dovetails nicely into conversations around [Bayesian inference](https://en.wikipedia.org/wiki/Bayesian_linear_regression), correlations, hypothesis testing, etc... There is a lot that falls out of thinking of LR from a statistics perspective.
+
+Formulating LR as a computer scientist would doesn't involve any of the above! It simply involves solving for a set of weights that minimizes the squared error between your estimate and the observation. No assumptions on Normality or independence or constant variance required. You end up with the same coefficients in both approaches, but now since you aren't making any assumptions on your data, you lose the ability to make inferences on the coefficients. This may or may not be useful to you, but, per the idea of learning the overlap, it's important to know that differences in these approaches exist.
+
+A similar example exists when learning logistic regression. The statisticians have their approach to formulating this problem which involves assumptions on residuals and objects called logit and GLMs. Computer scientists use words like cross-entropy loss and sigmoid. Both approaches get you to the same set of weights which minimize the same error function, but one approach allows you to calculate odds-ratios and confidence intervals around your coefficients and the other is a basic building block of learning about neural networks. Learn about how other fields formulate the same problem, in part to know how it works, and in part to be able to read the literature as you learn more. Data Science is built upon various fields each with their own problem formulation and terminology. Don't get confused.
+
+This stuff all builds on itself - don’t take short cuts. Understand what you’re doing and why. Learn the basics, but learn them well.
```

Comment:
>Understand what you’re doing and why.

I think we could provide more concrete tips on how to do this. What I normally say is that the best way to test understanding is to make predictions: "I think that in this model, if I change x, y will happen". Then I do it! If y happens, great, I understand what's going on, but _often_ y doesn't happen and I have to dig in and understand why. Doing this repeatedly is my favorite way to learn a new subject (which works especially well in programming domains where you can easily test your ideas with code!)

Reply:
Agreed I can be more clear on what I mean. Following the two examples of LR and Logistic Reg, I was more getting at understanding underlying assumptions in the models one uses. What assumptions are made, and why? What happens when assumptions are violated? What does one approach get you that the other doesn't? But what you suggest does get at the notion of understanding it empirically.